advancedDungeon/src/produsAdvanced/abstraction/Hero.java

```diff
@@ -64,6 +64,9 @@ public Hero(Entity heroInstance, FireballSkill fireballSkill) {
       pc.removeCallbacks(); // Entfernt alle bisherigen Tastenzuweisungen
     }
     this.fireballSkill = fireballSkill;
+    // uncap max hero speed
+    VelocityComponent vc = hero.fetch(VelocityComponent.class).get();
+    vc.velocity(Vector2.of(Integer.MAX_VALUE, Integer.MAX_VALUE));
```

Comment:
```suggestion
    hero.fetch(VelocityComponent.class).map(vc -> vc.velocity(Vector2.of(Integer.MAX_VALUE, Integer.MAX_VALUE)));
```

oder ähnlich (?)

Reply:
`vc -> vc.velocity(Vector2.of(Integer.MAX_VALUE, Integer.MAX_VALUE)` das dann in ne hilfsmethode

Reply:
@cagix gib mir nochmal streaming nachhilfe:

```java
 hero.fetch(VelocityComponent.class).map(Dummy::help);

...

private static VelocityComponent help(VelocityComponent vc )
 ```
 
 was mache ich, wenn ich `void help(VelocityComponent vc haben will?
 
 `hero.fetch(VelocityComponent.class)
    .ifPresent(Dummy::help);` soll man ja vermeiden, richtig?